.github/workflows/build_and_test.yml

```diff
@@ -126,11 +140,19 @@ jobs:
         docker_args+=(-e ENABLE_HYPRE_DEVICE=${ENABLE_HYPRE_DEVICE:-CPU})
         docker_args+=(-e ENABLE_TRILINOS=${ENABLE_TRILINOS:-ON})
 
-        docker_args+=(--cap-add=SYS_PTRACE)
+        docker_args+=(--cap-add=SYS_PTRACE --rm)
 
         script_args+=(--cmake-build-type ${{ inputs.CMAKE_BUILD_TYPE }})
         script_args+=(${{ inputs.BUILD_AND_TEST_CLI_ARGS }})
 
+        # SPLIT_DOCKER_REPOSITORY=(${DOCKER_REPOSITORY//// })
+        # CONTAINER_NAME=geosx_build_${SPLIT_DOCKER_REPOSITORY[1]}_${GITHUB_SHA:0:7}
+        # if [ "$(docker ps -aq -f name=${CONTAINER_NAME})" ]; then
+        #   docker rm -f ${CONTAINER_NAME}
+        # fi
+        # docker_args+=(--name ${CONTAINER_NAME})
```

Comment:
Please remove the commented attempts.

Reply:
Ahhh....I intentionally kept this in case we want to specify the container name and manual cleanup. It seems that adding the `--rm` argument to `docker run` takes care of the cleanup...BUT there are cases when we don't want to clean up. For instance if we remove the `--rm` the container remains and we can run the container from its saved state....which is handy for debugging.

Reply:
If you want to keep it, could you indicate your intent as a comment?

Reply:
Also, wouldn't it be simpler to remove the `--rm` argument if we want to keep the docker container?